Add start stop example (#4420)
influxdata/docs-v2: content/enterprise_influxdb/v1.10/administration/backup-and-restore.md

- `-start <timestamp>`: Include all points starting with specified timestamp (RFC3339 format). Not compatible with `-since` or `-strategy full`.
- `-end <timestamp>`: Exclude all points after timestamp (RFC3339 format). Not compatible with `-since` or `-strategy full`.

> #### Examples

Author: ```suggestion
### Examples
```

> ### Backup a database and all retention policies

Author: ```suggestion
#### Back up a database and all retention policies
```

influxd-ctl backup -db myfirstdb -rp autogen ./myfirstdb-autogen-backup
```

> ### Backup a database with a specific retention policy

Author: ```suggestion
#### Back up a database with a specific retention policy
```

influxd-ctl backup -db myfirstdb ./myfirstdb-allrp-backup
```

> ### Backup data from a specific time range

Author: ```suggestion
#### Back up data from a specific time range
```
